src/Scene/Scene.js

```diff
@@ -39,12 +39,17 @@ const RENDERING_ACTIVE = 1;
  * - remove debug boolean, replace by if __DEBUG__ and checkboxes in debug UI
  * - Scene (and subobjects) should be instanciable several times.
  */
-function Scene(crs, positionCamera, viewerDiv, debugMode, gLDebug) {
+function Scene(crs, coordinate, viewerDiv, debugMode, gLDebug) {
     if (instanceScene !== null) {
         throw new Error('Cannot instantiate more than one Scene');
     }
     this.referenceCrs = crs;
 
+    const positionCamera = coordinate.as(crs);
+    const coordinateTargetCamera = coordinate.clone();
+    coordinateTargetCamera._values[2] = 0;
```

Comment:
undescored properties are meant to be considered as private. You can't use it here. You either need to move this code or to add a function to Coordinate (`fromLonLatHeight` maybe)